classes/Formatter.php

```diff
@@ -56,21 +55,30 @@ private function get_issue_year( WP_Post $post ): string {
 	 * Get filtered and raw post content.
 	 */
 	private function get_post_content( WP_Post $post ): string {
-		$post_excerpt      = get_post_field( 'post_excerpt', $post );
-		$post_content      = get_post_field( 'post_content', $post );
-		$meta_content      = '';
-		$included_meta     = apply_filters( 'eep_meta_included_in_content', [] );
-		if ( ! empty( $included_meta ) ) {
-			foreach( $included_meta as $key ) {
-				$meta_content .= get_post_meta( $post->ID, $key, true );
+		$included_in_content = apply_filters( 'eep_included_in_content', [ 'post_field' => 'post_content' ] );
```

Comment:
Maybe I'm reading this wrong, but shouldn't it be an array of arrays, i.e. `'post_field' => [ 'post_content', ... ]` to be able to set more than one post_field, meta or taxonomy?

And then something like

```PHP
foreach ( $included_in_content as $type => $keys ) {
    foreach ( $keys as $key ) {
        switch ( $type ) {
       ....
```

Reply:
You are absolutly right. Did not cross my mind. 🥸